konfera/urls.py

```diff
@@ -18,4 +18,5 @@
 
 urlpatterns = [
     url(r'^event/(?P<event_slug>[\w,-]+)/$', views.sponsor_list_view, name='event_sponsors'),
+    url(r'^event/$', views.event_list, name='events')
```

Comment:
IMO add a comma to the end of the line so when someone adds a new `url` git doesn't show 3 changed instead of 1.